Server/src/models/SelectQueryDatabase.ts

```diff
@@ -0,0 +1,2806 @@
+import { EntityManager, getConnection } from "typeorm";
```

Comment:
This file needs to be split into maybe 2-3 smaller ones. Debugging 2000 line files can get tricky very fast